api/types/license.go

```diff
@@ -401,4 +419,6 @@ type LicenseSpecV3 struct {
 	SupportsMachineID Bool `json:"machine_id,omitempty"`
 	// SupportsResourceAccessRequests turns resource access request support on or off
 	SupportsResourceAccessRequests Bool `json:"resource_access_requests,omitempty"`
+	// IsTrial is true for trial licenses
+	IsTrial Bool `json:"is_trial,omitempty"`
```

Comment:
Can we rename this to simply `Trial`. We have a similar flag for `Cloud` (see a couple of lines above) and we don't call that one `IsCloud`. I'd like to keep the naming consistent.